stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -78,18 +80,21 @@ public type Client client object {
     #           may or may not continue to process the remaining commands in the batch. This property can be
     #           used to override this behavior. When it is set to true, if there is a failure in a few commands and
     #           the JDBC driver continues with the remaining commands, the successfully executed commands in the batch
-    #           also will get rolled back.
+    #           also will get rolled back
+    # + getGeneratedKeys - Allow to retrieves auto-generated keys after a statement has been executed if value is
+    #                      `true`.It will return auto-generated keys only if supported by JDBC
     # + return - A `BatchUpdateResult` with the updated row count and returned error if any. If all the commands
     #            in the batch have executed successfully, the error will be `nil`. If one or more commands have failed,
     #            the `returnedError` field will give the corresponding `Error` along with the int[] which
-    #            contains updated row count or the status returned from each command in the batch.
+    #            contains updated row count or the status returned from each command in the batch
     public remote function batchUpdate(@untainted string sqlQuery, boolean rollbackAllInFailure,
-                                       Param?[]... parameters)
+                                       boolean? getGeneratedKeys = (), Param?[]... parameters)
```

Comment:
It seems the parameters are not consistent in this remote function now. The `rollbackAllInFailure` is a `boolean` while the next similar parameter `getGeneratedKeys` is `boolean?`